index.d.ts

```diff
@@ -0,0 +1,41 @@
+declare module 'ember-shepherd/services/tour' {
+  import Shepherd from 'shepherd.js';
+
+  export interface TourButton extends Shepherd.Step.StepOptionsButton {

```

Comment:
This interface isn't really part of the tour service, but is needed because the default Shepard.js button type does not have a `type` property